app.js

```diff
@@ -63,17 +63,17 @@ if (opts.debug) logger.level = 'debug';
 
 // AP: Detect if we're running under Windows Subsystem for Linux. Temporary modification
 // of process.env is allowed: https://nodejs.org/api/process.html#process_process_env
-if ((process.platform === "win32") || child_process.execSync('uname -a').toString().indexOf('Microsoft') > -1)
+if (process.platform === "linux" && child_process.execSync('uname -a').toString().indexOf('Microsoft') > -1) {
```

Comment:
It only just hit me why you changed this, and I'm happy you did. I encountered so many issues with having env.wsl on true while on Win32. Excellent 👍